tests/smoke/support/page/runtime.ts

```diff
@@ -9,7 +9,8 @@ export class RuntimePage {
 
   async navigateToApp({ name }: { name: string }): Promise<void> {
     const { page } = this.actor
-    await page.click('#_appSwitcherButton')
```

Comment:
I think we can stay with the current solution in line 12 - no need for the double await, no?

Reply:
we get failure if use `await page.click('#_appSwitcherButton')` We need use here waitForSelector.
`const element = await page.waitForSelector('#_appSwitcherButton')
    element.click()` solution works